src/wxProfilerGUI/CallstackView.cpp

```diff
@@ -153,7 +148,9 @@ void CallstackView::showCallStack(const Database::Symbol *symbol)
 	}
 
 	callstacks = database->getCallstacksContaining(symbol);
-	std::sort(callstacks.begin(),callstacks.end(),SortCalls);
+	std::sort(callstacks.begin(), callstacks.end(), [this](const Database::CallStack *a, const Database::CallStack *b) {
+		return database->getFilteredSampleCount(a->samples) > database->getFilteredSampleCount(b->samples);
+	});

```

Comment:
:warning: I believe this change increases the complexity of the sort from O(n log n) to O(n^2 (log n) (log n)). I'm concerned that this may become a problem on larger profiling results.

Reply:
Would it make sense to do a Schwartzian transform here?

Reply:
I did that, it should address any potential slowdown there.